Update copy on quick-start (#106)
storyscript/docs.storyscript.io: quick-start/README.md

First, install our CLI:

<table width="100%">
<tr>
<td style="text-align:center" width="100%" valign="top" colspan="2">
<details :open="$page.os === 'macos'">
<summary><h4><img src="../assets/apple-logo.svg" width="15"> macOS</h4></summary>

```bash
brew install storyscript/brew/story
```

</details>
</td>
<!--
<td style="text-align:center" width="50%" valign="top">
<details :open="$page.os === 'windows'">
<summary><h4><img src="../assets/windows-logo.svg" width="15"> Windows</h4></summary>

Download the appropriate installer:

<div><a href="https://github.com/asyncy/cli/releases/download/0.0.6/asyncy-x64.exe" class="button">64-bit installer</a></div>
<div><a href="#" class="button">32-bit installer</a></div>

</details>
</td>
-->
</tr>
<tr>
<!--
<td style="text-align:center" width="50%" valign="top">
<details :open="$page.os === 'unix' || $page.os === 'linux'">
<summary><h4><img src="../assets/ubuntu-logo.svg" width="15"> Ubuntu 16+</h4></summary>

<a href="https://snapcraft.io/story">
  <img alt="Get it from the Snap Store" src="https://snapcraft.io/static/images/badges/en/snap-store-white.svg" />

> </a>

Reviewer: Why not show the `sudo snap install story`?

![image](https://user-images.githubusercontent.com/4370550/60434924-91205f00-9c08-11e9-8467-f0d7086481fe.png)

See: https://docs.storyscript.io/dev/cli/#install

Reviewer 2: Yea, let's add back snap, it works now.

<small style="display:block; width: 100%"><a href="https://snapcraft.io/">Snap is available on other Linux OS.</a></small>

</details>
</td>
-->
<td style="text-align:center" width="50%" valign="top">
<details :open="$page.os === 'unknown'">
<summary><h4>Direct from Python</h4></summary>

```bash
pip install --user story
```

Python 3.6 or higher is required, thus on Debian/Ubuntu use `pip3`.
The other installation methods listed are recommended.

</details>
</td>
</tr>
</table>

Next, login with your GitHub account:

```bash
story login
```

All done! 💪 You're all set to create and deploy your apps.

## Write your first Story

Let's create your first application. First, create a new folder for your application.

```bash
mkdir first_app && cd first_app
```
```bash

> story apps create

Reviewer: @judepereira that's one of the commands I was talking about at the team summit. Imho we should add a global alias, s.t. it's just `story create`.

What do you think? I opened an issue -> https://github.com/storyscript/cli/issues/224

Reviewer 2: I like it :)

```

> > This generated the `story.yml` file which is used to identify your application and provide metadata to configure version and secrets variables.

Reviewer: ```suggestion
> This command generated a `story.yml` file which is used to identify your application and provide metadata to configure version and secrets variables.
```

We have created a few examples that can help you bootstrap your project. Let's start you off with the classic hello-world:

```bash

> story write http > http.story

Reviewer: Slight OT: Can we rename this to `example` or `templates`
I opened an issue -> https://github.com/storyscript/cli/issues/225

```

Let's take a look:

```bash
cat http.story
```

```coffeescript
when http server listen method: 'get' path: '/' as request
    request write content: 'Hello world!'
```
> It's just a simple http server but a great starting point to just get *something* working. We have many other simple and complex examples [here](https://github.com/storyscript/examples).

## Deploy to Storyscript Cloud

> The Storyscript Cloud is a microservice and function platform that uses Kubernetes and other cloud-native tooling as a fully managed microservice orchestration runtime. Simple put, **Storyscript make it easy to deploy and use microservices on Kubernetes.** 💪

Reviewer 3: ```suggestion
The Storyscript Cloud is a microservice and function platform that uses Kubernetes and other cloud-native tooling as a fully managed microservice orchestration runtime. Simple put, **Storyscript makes it easy to deploy and use microservices on Kubernetes.** 💪
```

Let's deploy your story:

```bash
story deploy
```

🎉 Congratulations! You have just deployed your first Story!

::: tip What just happened behind the scene?

The output from your deployment will share some details about what happened, because it's quite amazing!

1. Storyscript is **strongly-typed**, therefore your stories were verified and syntax checked before deploying.
1. Services are pulled from the [Hub](https://hub.storyscript.io) and deployed to Kubernetes pods.
1. *Coming soon!* Functions were packaged in containers and deployed as services like above.

> 1. Services were started, events subscribed and ingress routes assigned.

Reviewer: Don't we call them `exposes` (https://docs.storyscript.io/reference/exposing_services/) instead of ingress routes?

Reviewer 2: No, we call them forwards.

1. The Storyscript Runtime caches your story for fast execution, acting as a microservice orchestrator.

The above would have mostly been manual work, typically in the form of Kubernetes configuration files which are automatically generated during deployment in the Storyscript Cloud.
:::

## Putting it all together

A quick introduction to how the Storyscript platform works.

> The **Stroyscript language** is a new language designed for business-logic which describes the flow of data between services. It **does not replace any other language**; is unites all other languages in a polyglot development platform being the most inclusive programming language. The language was inspired by many other languages giving it a natural and intuitive familiarity. It's **strongly-typed** and compiled into the Storyscript cloud-native runtime. We call this **top-level coding with a cloud-native programming language**.

Reviewer 3: ```suggestion
The **Stroyscript language** is a new language designed for business-logic which describes the flow of data between services. It **does not replace any other language**; it unites all other languages in a polyglot development platform being the most inclusive programming language. The language was inspired by many other languages giving it a natural and intuitive familiarity. It's **strongly-typed** and compiled into the Storyscript cloud-native runtime. We call this **top-level coding with a cloud-native programming language**.
```
